### DISCUSSION TITLE
Sorting Related Columns in Content Search Page

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#13579] [Archive]
Hi,

I have this collection that has a translations field. I see the same behavior with other M2A fields as well.

In the content search page, the related fields, in this case Title and Text, cannot be sorted. Those two fields were part of the collection previously but I have moved them into the translations. 

![image](https://user-images.githubusercontent.com/7398499/170531744-316f77b9-5193-4a4c-83e0-8a4c026fdcdc.png)

**Comment by rijkvanzanten** [comment]
You can't sort on o2m type values, as they're arrays of values rather than a single value to sort on. What would you expect sort to do in this case? Sort by the number of related items? Sort by the first item in the array of related items?

**Reply by st11x** [reply]
I suppose this is a special case of the o2m because the field is a translation, and you will ever only see 1 value through the user interface.  So I was expecting to be able to sort by the translated Title as if the Title belonged to the Paragraph collection itself.

**Reply by rijkvanzanten** [reply]
That depends on the display that's configured, but I know what you're saying!
